NUModules/NUModule.j

```diff
@@ -3339,7 +3417,16 @@ NUModuleTabViewModeIcon                = 2;
     _categories = someCategories;
     [self didChangeValueForKey:@"categories"];
 
-    _usesPagination = ![_categories count];
+    // Use pagination only if all categories specify a context
+    _usesPagination = YES;
+    for (var i = 0; i < [_categories count]; i++)
+    {
+        if (![_categories[i] contextIdentifier])
+        {
+            _usesPagination = NO;
+            break;
+        }
+    }
```

Comment:
I think we should force the category to have a fetcher, no ?
If we agree, let's throw an error to ensure all categories have a fetcher and we will fix our current code